easybuild/easyconfigs/c/Cartopy/Cartopy-0.22.0-foss-2023b.eb

```
@@ -0,0 +1,56 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Updated: Denis Kristak
+easyblock = 'PythonBundle'
+
+name = 'Cartopy'
+version = '0.22.0'
+
+homepage = 'https://scitools.org.uk/cartopy/docs/latest/'
+description = """Cartopy is a Python package designed to make drawing maps for data analysis and visualisation easy."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    # sufficiently recent Cython is required, to fix issues like:
+    #   pykdtree/kdtree.c:2437:76: error: PyArrayObject {aka struct tagPyArrayObject} has no member named data
+    # Cython included with Python-bundle-PyPI (0.29.35) is not sufficient
+    ('Cython', '3.0.10'),
+    ('Fiona', '1.9.5'),
```

Use the one from #22070:
```suggestion
    ('Fiona', '1.10.1'),
```